docs/book/advanced-config.md

```diff
@@ -36,15 +36,9 @@ return [
 
     // These are various options for the listeners attached to the ModuleManager
     'module_listener_options' => [
-        // This should be an array of paths in which modules reside.
-        // If a string key is provided, the listener will consider that a module
-        // namespace, the value of that key the specific path to that module's
-        // Module class.
-        'module_paths' => [
-            './module',
-            './vendor',
-        ],
-
+        // use composer autoloader instead of laminas-loader
+        'use_laminas_loader' => true,
```

Comment:
I bet you mean `false`.

See also:
https://github.com/laminas/laminas-mvc-skeleton/blob/master/config/application.config.php#L15-L16